cmd/notation/policy/import.go

```diff
@@ -65,7 +65,7 @@ func runImport(command *cobra.Command, opts importOpts) error {
 			}
 		}
 	} else {
-		fmt.Fprintf(os.Stderr, "Warning: existing trust policy configuration file will be overwritten")
+		fmt.Fprintf(os.Stderr, "Warning: existing trust policy configuration file will be overwritten\n")
```

Comment:
Can we use `Fprintln` instead?

Reply:
Yeah, that will also work. Updating the code.